enhancements/api-review/custom-tags-aws.md

```diff
@@ -48,62 +50,147 @@ Motivations include but are not limited to:
 - the administrator or service (in the case of Managed OpenShift) installing OpenShift can pass an arbitrary
    list of user-defined tags to the OpenShift Installer, and everything created by the installer and all other
    bootstrapped components will apply those tags to all resources created in AWS, for the life of the cluster, and where supported by AWS.
-- tags must be applied at creation time, in an atomic operation. It isn't acceptable to create an object and,
-   after some period of time, apply the tags post-creation.
+- tags can be applied at creation time, in an atomic operation.
+- tags can be updated post creation.
 
 ### Non-Goals
 
-- to reduce initial scope, tags are applied only at creation time and not reconciled. If an administrator manually
-   changes the tags stored in the infrastructure resource, behavior is undefined. See below.
 - to reduce initial scope, we are not implementing this for clouds other than AWS. We will not take any actions
    to prohibit that later.
 
+### Limitations
+
+- tags for PV volumes cannot be updated but only set during creation.
```

Comment:
maybe worth calling out why. I personally would love to see this solved upstream, eg through:
 
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/1014 
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/180#issuecomment-947568029